.travis.yml

```diff
@@ -35,23 +34,23 @@ before_install:
   - export PATH="$HOME/.yarn/bin:$PATH"
 
 install:
-  - yarn install --production=false
+  - yarn install
 
 jobs:
   include:
   - stage: pull_request
     script:
       - >-
-        yarn concurrently
-        "yarn lint"
-        "yarn depcheck"
-        "yarn test"
+        yarn run-p -l
+        lint
+        depcheck
+        test
       - >- # Chromatic and Cypress rely on Storybook being built, this will exit early if build-storybook fails
         yarn build-storybook &&
-        yarn http-server docs -p 9001 & yarn wait-on http://localhost:9001 &&
-        yarn concurrently
-        "yarn chromatic --quiet --exit-zero-on-changes --storybook-build-dir docs"
-        "yarn cypress run --record" &&
+        yarn http-server docs -p 9001 & npx wait-on http://localhost:9001 &&
+        yarn run-p -l
+        "chromatic -- --quiet --exit-zero-on-changes --exit-once-uploaded --storybook-build-dir docs"
```

Comment:
Most of the work is done on Chromatic's side so this allows us to make our Chromatic build command exit after we've uploaded the Storybook source. This also ensures that we don't incur CI build time increases in Travis as we add more visual testing stories.

Reply:
```
--exit-once-uploaded
```

Genius. I didn't matter much when we only had Chrome and it took like 15s for 300 stories. IE11 is much slower. We already have a status check that is reported by Chromatic. We don't need to wait for it here. 👍